main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Inside of `migration.Migrate()` everything is done in a transaction. So potentially two cores should not run the same migration twice.